src/community/ogcapi/ogcapi-features/src/main/java/org/geoserver/ogcapi/v1/features/FeaturesAPIBuilder.java

```diff
@@ -68,7 +86,17 @@ public OpenAPI build(WFSInfo wfs) throws IOException {
 
         // list of valid filter-lang values
         Parameter filterLang = parameters.get("filter-lang");
-        filterLang.getSchema().setEnum(new ArrayList<>(APIFilterParser.SUPPORTED_ENCODINGS));
+        ArrayList<String> filterLangValues = new ArrayList<>(APIFilterParser.SUPPORTED_ENCODINGS);
```

Comment:
Based on configuration, the sortby, filter, filter-lang parameters should also be removed (and so on)

Reply:
path: search, queryables and functions now removable

parameters: filter, filter-lang, sortBy now removable